content/terminal/openbb-copilot.md

```diff
@@ -5,215 +5,340 @@ description: Learn how to use OpenBB Copilot to interact with the OpenBB Termina
+
+:::note
+
+The newly-created widget contains all of the data in the artifact, so you can
+interact with it just like any other widget on your dashboard (including
+switching between different types of charts, or toggling between chart and table
+views, or adding it as explicitly-added context to OpenBB Copilot for further
+analysis).
+
+:::
+
+## Searching the web
 
-### Artifacts
+OpenBB Copilot has the ability to search the web. 
 
-A special kind of status update is the result artifact which has the final data used to answer the query. It either be a text or a table.
-The contents of the artifact are produced by the reasoning of the OpenBB Copilot.
+To search the web, use the `@web` keyword in your query to tell OpenBB Copilot
+to search the web for information. For example:
 
 <div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/artifact_chat_example.jpg" alt="result artifact openbb copilot" width="80%" height="80%" />
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/web_search_example.png" alt="web search openbb copilot" width="60%" height="60%" />
 </div>
 
-To ensure transparency and extend the analysis, OpenBB Copilot will also return intermediate result artifacts that were used as reasoning steps to arrive to the final artifact and thus the answer.
+## Reasoning Steps
+As OpenBB Copilot goes through the process of answering your query, it will show
+you the reasoning steps it took to arrive at the answer.  These reasoning steps
+appear above Copilot's responses. Reasoning steps can be expanded to see more
+information about the intermediate steps OpenBB Copilot took to arrive at the
+answer:
 
 <div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/intermediate_result_artifact_chat_example.jpg" alt="intermediate result artifact openbb copilot" width="80%" height="80%" />
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/expanded_reasoning_steps.png" alt="expanded reasoning steps openbb copilot" width="60%" height="60%" />
 </div>
 
-## Creating widgets from OpenBB Copilot responses
+The reasoning steps can contain parameters for data sources, internal queries,
+which files are accessed, intermediate artifacts, and more.
+
+## Citations
 
-Each time OpenBB Copilot generates a response, it will have a "Create widget from text" or "Create widget from table" button. Clicking this button will create a new widget on your dashboard based on the response.
+When accessing data (whether from widgets on your dashboard, files you have
+uploaded, or external data sources), OpenBB Copilot will always cite the data
+source it referenced when responding to your query:
+
+<div style={{display: 'flex', justifyContent: 'center'}}>
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/citations.png" alt="citations openbb copilot" width="80%" height="80%" />
```

Comment:
I'm getting the `alt`, this image is not available.

<img width="409" alt="Screenshot 2024-12-04 at 11 10 19" src="https://github.com/user-attachments/assets/a72d24f0-8ecd-4f00-a28d-69c6d3c000a5">

Reply:
Fixed.